src/ts/components/account/AccountDropdown.tsx

```diff
@@ -157,6 +161,13 @@ class AccountDropdown extends React.Component<IDashboardProps, IDashboardState>
     }
   }
 
+  componentDidMount () {
+    if (this.props.settings.selectedAccount) {
+      const account = this.props.settings.selectedAccount
+      this.props.getTransactions(account.address)
```

Comment:
Do you still need to load transactions here? You seem load transactions in transaction list component already

Reply:
Because `Send` page also use this component, this ensures `Send` page loads the latest transaction list for the selected account. That latest transaction list can then be updated and saved during and after the transaction.

Reply:
In this case, we will load transactions when dropdown account is selected on every page where dropdown is placed .

Reply:
yes, it could be a performance drawback if the account contains a hug list of transaction history. But generally speaking, I don't expect people do thousands of transactions via the extension.